blackjack.py

```diff
@@ -79,43 +88,37 @@ def print_entire_hand(self):
         for card in self.hand:
```

Comment:
Commenting about the code above.
The method `add_card_to_hand` isn't bad, but it seems like we should be able to do better, I'm sure Python has a way to help us here.

Since `Player` has only one attribute, `hand` which is a `list`, one could say that a `Player` can be represented by a `list`. So why not try to inherit from it?
```
class Player(list):
    pass
```
What can we do with that? Well let's see.
```
In [2]: player = Player()

In [3]: player
Out[3]: []

In [4]: player.append(1)

In [5]: player
Out[5]: [1]

In [6]: player.append(2)

In [7]: player
Out[7]: [1, 2]
```

So we wouldn't need a `add_card_to_hand` method anymore!
Some work would need to be done for it to work with the rest of your code, but it could be a good exercise.

Continuing with `print_entire_hand`, you should have used `__repr__` like you've done in `Card` instead, it'll allow you to run:
```
print(player)
```

Reply:
@romainrichard 
In relation to the last comment, should I replace __repr__ with __str__ and use it in card and for print_entire_hand ? 
```
__repr__(self) - Defines behavior for when repr() is called on an instance of your class. The major difference between str() and repr() is intended audience. repr() is intended to produce output that is mostly machine-readable (in many cases, it could be valid Python code even), whereas str() is intended to be human-readable.
```

Reply:
@romainrichard In relation to changing player to inherit from list, Would this cause issues due to the fact that Player is the Parent Class or Gambler and Dealer?

Reply:
You can.
Personally I never had to go as far as needing to define both `__repr__` and `__str__` so I always to `__repr__` and consider it done. But you want to practice so do both :slightly_smiling_face: